functions/index.js

```diff
@@ -142,8 +142,8 @@ exports.createCheckoutSession = functions.https.onCall(async (data, context) =>
         subscription_data: {
             items: data.items
         },
-        success_url: 'https://bythebookthebible.com/memorize',
-        cancel_url: 'https://bythebookthebible.com/subscribe',
+        success_url: "https://schmudgin.bythebookthebible.com",
```

Comment:
Is there a way to programmatically retrieve the URL instead of hard-coding it? Might be tricky since we're inside a cloud function, not on the main page itself. If there is a way to retrieve it on the main page, we could pass it in as a parameter to createCheckoutSession.